CHANGELOG.md

```diff
@@ -1,3 +1,238 @@
+
+* Allows disabling nodes when multiple failures happen
+
+* Delegates function calls of the form `SELECT .. FROM func()`
+
+* `citus_shard_indexes_on_worker` shows all local shard indexes regardless of
+  `search_path`
+
+* `citus_shards_on_worker` shows all local shards regardless of `search_path`
+
+* Deprecates inactive shard state, never marks any placement inactive
+
+* Disables distributed & reference foreign tables
+
+* Prevents propagating objects having a circular dependency
+
+* Prevents propagating objects having a dependency to an object with unsupported
+  type
+
+* Deprecates `master_get_table_metadata` UDF
+
+* Allows users to add local tables to metadata using
+  `citus_add_local_table_to_metadata()` UDF
+
+* Disallows remote execution from queries on shards
+
+* Drops `citus.enable_cte_inlining` GUC
+
+* Drops `citus.single_shard_commit_protocol` GUC, defaults to 2PC
+
+* Drops support for `citus.multi_shard_commit_protocol`, always use 2PC
+
+* Avoids unnecessary errors for `ALTER STATISTICS IF EXISTS` when the statistics
+  does not exist
+
+* Fixes a bug that causes columnar storage pages to have zero LSN
+
+* Fixes a bug that causes issues while create dependencies from multiple
+  sessions
+
+* Fixes a bug that causes reading columnar metapage as all-zeros when
+  writing to a columnar table
+
+* Fixes a bug that could break `DROP SCHEMA/EXTENSON` commands when there is a
+  columnar table
+
+* Fixes a bug that could break pg upgrades due to missing `pg_depend` records
+  for columnar table access method
+
+* Fixes a bug that could cause `CREATE INDEX` to fail for expressions when using
+  custom `search_path`
+
+* Fixes a bug that could cause `worker_save_query_explain_analyze` to fail on
+  custom types
+
+* Fixes a bug that could cause failed re-partition joins to leak result tables
+
+* Fixes a bug that could cause prerequisite columnar table access method
+  objects being not created during pg upgrades
+
+* Fixes a bug that could cause re-partition joins involving local shards to fail
+
+* Fixes a bug that limits usage of sequences in non-int columns
+
+* Fixes a bug that prevents `DROP SCHEMA CASCADE`
+
+* Fixes a build error that happens when `lz4` is not installed
+
+* Fixes a clog lookup failure that could occur when writing to a columnar table
+
+* Fixes a crash that occurs when the aggregate that cannot be pushed-down
+  returns empty result from a worker
+
+* Fixes a missing `FROM` clause entry error
+
+* Fixes a possible segfault that could happen when reporting distributed
+  deadlock
+
+* Fixes an issue that could cause unexpected errors when there is an in-progress
+  write to a columnar table
+
+* Fixes an unexpected error that occurs due to aborted writes to a columnar
+  table with an index
+
+* Fixes an unexpected error that occurs when writing to a columnar table created
+  in older version
+
+* Improves handling of `IN`, `OUT` and `INOUT` parameters for functions
```

Comment:
Minor: maybe move this to the bottom of `supported` entries.

Initially the wording of this also was 'fixes`, but given that it was never supported due the the bug, I see this as a new feature.